test/controllers/api/v2/trends_controller_test.rb

```diff
@@ -0,0 +1,69 @@
+require 'test_helper'
+
+class Api::V2::TrendsControllerTest < ActionController::TestCase
+  foreman_trend_valid_attrs = { :trendable_type => 'Model' }
```

Comment:
Just for the sake of uniformity, could you place this line inside `setup` block as class variable just as you do with `@fact_trend_valid_attrs`?